de.blobbyvolley.BlobbyVolley2.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally should be submitted upstream as well

Reply:
Yep, upstream developer will decide.

Reply:
https://github.com/danielknobe/blobbyvolley2/issues/16

Reply:
Thanks!